kview/src/main/java/dev/ikm/komet/kview/mvvm/view/properties/StampAddController.java

```diff
@@ -0,0 +1,142 @@
+package dev.ikm.komet.kview.mvvm.view.properties;
+
+import dev.ikm.komet.kview.mvvm.view.AbstractBasicController;
+import dev.ikm.komet.kview.mvvm.viewmodel.StampViewModel2;
+import dev.ikm.tinkar.common.util.text.NaturalOrder;
+import dev.ikm.tinkar.entity.ConceptEntity;
+import dev.ikm.tinkar.terms.EntityFacade;
+import dev.ikm.tinkar.terms.State;
+import javafx.beans.property.ObjectProperty;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.ComboBox;
+import org.carlfx.cognitive.loader.InjectViewModel;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static dev.ikm.komet.kview.mvvm.viewmodel.StampViewModel2.StampProperties.MODULE;
+import static dev.ikm.komet.kview.mvvm.viewmodel.StampViewModel2.StampProperties.MODULES;
+import static dev.ikm.komet.kview.mvvm.viewmodel.StampViewModel2.StampProperties.PATH;
+import static dev.ikm.komet.kview.mvvm.viewmodel.StampViewModel2.StampProperties.PATHS;
+import static dev.ikm.komet.kview.mvvm.viewmodel.StampViewModel2.StampProperties.STATUS;
+import static dev.ikm.komet.kview.mvvm.viewmodel.StampViewModel2.StampProperties.STATUSES;
+
+
+public class StampAddController extends AbstractBasicController {
```

Comment:
Can we not extends `extends AbstractBasicController`?
We are using MVVM and no need to use methods to update ui.